### PR TITLE
Fix view debug panel with. 2.0.10 version

### PR DESCRIPTION
**Pull request by drdim**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | no

Can view debug panel with latest base debug panel yii styling class

### CODE REVIEW

**Comment by samdark** [comment]
Need a line for CHANGELOG.

File: DebugPanel.php [hunk 1]
```diff
@@ -56,9 +56,9 @@ public function getSummary()
         $queryTime = number_format($queryTime * 1000) . ' ms';
         $url = $this->getUrl();
         $output = <<<EOD
-<div class="yii-debug-toolbar-block yii-debug-toolbar__block">
+<div class="yii-debug-toolbar__block">
```
Comment (beowulfenator) [inline DebugPanel.php]: This is good.

File: DebugPanel.php [hunk 2]
```diff
@@ -56,9 +56,9 @@ public function getSummary()
...
     <a href="$url" title="Executed $queryCount elasticsearch queries which took $queryTime.">
-        ES <span class="label yii-debug-toolbar__label yii-debug-toolbar__label_info">$queryCount</span> <span class="label yii-debug-toolbar__label">$queryTime</span>
+        ES <span class="yii-debug-toolbar__label yii-debug-toolbar__ajax_counter yii-debug-toolbar__label_success">$queryCount</span> <span class="yii-debug-toolbar__label yii-debug-toolbar__label_info">$queryTime</span>
```
Comment (beowulfenator) [inline DebugPanel.php]: This is debatable. The original DB component assigns the blue info class to the query count and no class to query time (making query time grey). Perhaps we should do the same here?

**Comment by beowulfenator** [comment]
Actually, now i remember why the "yii-debug-toolbar-block" class is there. Last time I kept it for compatibility with older yii2 versions, but i guess there's no point doing that now.

File: DebugPanel.php (hunk 2 above)
Comment (drdim) [inline DebugPanel.php]: change color with Database Queries

**Comment by samdark** [comment]
Looks good to me. @beowulfenator merge?

**Pull request merged**